### DISCUSSION TITLE
Add ability to defer loading of livewire javascript

### DISCUSSION THREAD

**Discussion by tdondich** [livewire/livewire#5177] [3. Feature ideas]
For performance reasons, it would be great to load the livewire script with defer, or at least have it as an option.  Passing in defer=true into options and then pasisng it into LivewireManager::javascriptAssets would be great!  Let me know if interested and I could produce a PR if needed.

**Comment by joshhanley** [comment]
@tdondich your Livewire scripts should be at the end of the body tag, so they don't run until last anyway so shouldn't need to be deferred. Or do you mean something different?

**Reply by tdondich** [reply]
The problem is it blocks rendering technically when still in the body.  By adding defer, along with other scripts, we optimize the loading and don't get penalized by core web vitals.
